### DISCUSSION TITLE
使用SCSS 变量修改主题，希望在某个页面下的组件为自定义主题，但是非该页面的组件还是原先的主题，请问能实现吗？

### DISCUSSION THREAD

**Discussion by ghost** [element-plus/element-plus#13276] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `All`

## Description

希望在项目的某个部分使用自定义主题，而不是整套项目使用自定义主题。如我的项目结构分左中右三个区域，仅仅希望中间区域应用自定义主题。

### Additional comments

简单示例demo的gitgub地址：https://github.com/kimxu97/demo

**Comment by kaine0923** [comment]
我觉得你可以用Config Provider 配合namespace来实现。在中间的组件包裹在el-config-provider内
